### PR TITLE
add Scala.js/Scala Native/Android to community page

### PR DESCRIPTION
**Pull request by SethTisue**
fyi @sjrd @densh @charafau

### CODE REVIEW

**Comment by heathermiller** [comment]
Nice :) Thanks @SethTisue, LGTM

File: community/index.md [hunk 1]
```diff
@@ -113,3 +113,9 @@ Staying current:
 
 * [implicit.ly](http://implicit.ly) announces new versions of Scala libraries as they become available
 * [Scala-Announce](http://groups.google.com/group/scala-announce) also has release announcements
+
+## Non-JVM platforms
+
+* [Scala.js](http://www.scala-js.org) compiles Scala code to JavaScript
```
Comment (sjrd) [inline community/index.md]: May I suggest http*s*? We do serve https://www.scala-js.org/

**Comment by sjrd** [comment]
👍

**Comment by sjrd** [comment]
LGTM

**Comment by charafau** [comment]
nice! thank you Seth.

**Pull request merged**